### DISCUSSION TITLE
ToggleSwitch styling from 0.0.10 to 11.0.0

### DISCUSSION THREAD

**Discussion by revenz** [AvaloniaUI/Avalonia#12610] [Q&A] [answered]
In version 0.0.10, I could style the toggle switch like so

```
    <Application.Resources>
        <SolidColorBrush x:Key="ToggleSwitchFillOn" Color="#ed1a3d" />
        <SolidColorBrush x:Key="ToggleSwitchFillOnPointerOver" Color="#ed1a3d" />
        <SolidColorBrush x:Key="ToggleSwitchFillOnPressed" Color="#ed1a3d" />
        <SolidColorBrush x:Key="ToggleSwitchStrokeOn" Color="#ed1a3d" />
        <SolidColorBrush x:Key="ToggleSwitchStrokeOnPointerOver" Color="#ed1a3d" />
        <SolidColorBrush x:Key="ToggleSwitchStrokeOnPressed" Color="#ed1a3d" />
        <SolidColorBrush x:Key="ToggleSwitchStrokeOnDisabled" Color="#ed1a3d" />
    </Application.Resources>
```

This doesn't work in 11.   

How can I change it in version 11?  Couldnt find this on the docs or digging through the source.

Also the animation seem to be gone, it just jumps on/off when clicked without sliding.

I'm using the Simple theme

TIA

**Comment by timunie** [comment]
~~here is the original style:~~ ~~https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Themes.Fluent/Controls/ToggleSwitch.xaml~~ 

~~check if your keys still matches.~~ 

wrong comment. See below.

**Reply by timunie** [reply]
sorry, you mentioned simple theme. The keys you override are from fluent theme. Here is simple theme: https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Themes.Simple/Controls/ToggleSwitch.xaml

**Comment by revenz** [comment, marked as answer]
Thanls. this did what i wanted

```
        <Style Selector="ToggleSwitch:checked /template/ Border#SwitchKnobBounds">
            <Setter Property="Background" Value="#ed1a3d"></Setter>
        </Style>
```
